server_environment/models/server_env_mixin.py

```diff
@@ -332,8 +332,14 @@ def _server_env_transform_field_to_read_from_env(self, field):
         field.compute = "_compute_server_env"
 
         inverse_method_name = "_inverse_server_env_%s" % field.name
-        inverse_method = partialmethod(type(self)._inverse_server_env, field.name)
-        setattr(type(self), inverse_method_name, inverse_method)
+        inverse_method = partial(
+            type(self)._inverse_server_env, self=self, field_name=field.name
+        )
+        setattr(
+            type(self),
+            inverse_method_name,
+            update_wrapper(inverse_method, type(self)._inverse_server_env),
```

Comment:
hmm weird... I've tried the same fix w/ update_wrapper but it didn't work...
Ah but you are not using `partialmethod` anymore.... :thinking:

Reply:
Also, this solution won't set the right name but the name of the wrapped func. I tried overriding `__name__` but that won't work, at least on v14

Reply:
Works well in v16.0
![image](https://user-images.githubusercontent.com/8638295/229729461-33f7d6de-b3d9-4b82-854c-7f1084388138.png)